src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -374,14 +379,15 @@ public void fireFileStarted(String fileName) {
      * Notify all listeners about the errors in a file.
      *
      * @param fileName the audited file
-     * @param errors the audit errors from the file
+     * @param fileProcessingResult file processing results object
      */
     @Override
-    public void fireErrors(String fileName, SortedSet<LocalizedMessage> errors) {
+    public void fireErrors(String fileName, FileProcessingResult fileProcessingResult) {
```

Comment:
@rnveach 
Yes, we can get the name of the file from FileText which will become part of FileProcessingResult. Thus, we do not need the first parameter of ```fireErrors``` at all. 

@timurt 
Please, change the signature of fireErrors to
```java
void fireErrors(FileProcessingResult fileProcessingResult) 
```